frontend/src/app/main/ui/comments.cljs

```diff
@@ -381,7 +382,7 @@
                        (some? position-modifier)
                        (gpt/transform position-modifier))
 
-        max-height   (int (* (:height viewport) 0.75))
+        max-height   (when (some? viewport) (int (* (:height viewport) 0.75)))
```

Comment:
having the `or` on the line 359 for viewport, I guess this `some?` condition looks redundant

Reply:
Hi! Not exactly, without this change then `max-height` is set to `0` and not `nil`, and then it gets added to the element styles as `max-height: 0px` and the fix does not work (i.e. we need this property to disappear from the stylesheet altogether).

Reply:
Ah now I see what you mean. The problem is that GitHub is omitting some of the lines and it seems that the diff shows the same function, but they are two different functions, so the `viewport` var is different in each one.